sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -620,7 +622,7 @@ abstract class CodeGenerator[InType <: AnyRef, OutType <: AnyRef] extends Loggin
    * expressions that don't support codegen
    */
   def newCodeGenContext(): CodegenContext = {
-    new CodegenContext
+    new CodegenContext(new SimpleCodegenConf)
```

Comment:
I overlooked the purpose of this function. It's for expressions that don't support codegen. I replaced `SimpleCodegenConf` with `EmptyCodegenConf`.
```
/**
   * Create a new codegen context for expression evaluator, used to store those
   * expressions that don't support codegen
   */
```